webapp/channels/src/components/widgets/menu/menu_items/menu_cloud_trial.test.tsx

```diff
@@ -110,10 +110,10 @@ describe('components/widgets/menu/menu_items/menu_cloud_trial', () => {
                     },
                 },
                 cloud: {
-                    customer: null,
-                    subscription: null,
-                    products: null,
-                    invoices: null,
+                    customer: undefined,
```

Comment:
Nit: Can we just remove these?

Reply:
Yeah, i can do that. I think I was worried about there being a difference if I removed them similar to my other comment, but it seems harmless since this is just in the test code